openedx/features/colaraz_features/helpers.py

```diff
@@ -498,3 +506,57 @@ def add_user_fullname_in_threads(threads):
             add_user_fullname_in_threads(thread.get('endorsement'))
 
     return threads
+
+
+def create_oauth2_client_for_ecommerce_site(url, site_name, service_user):
+    """
+    Creates the oauth2 client and add it in trusted clients.
+    """
+    client, created = Client.objects.get_or_create(
+        **{
+            "user": service_user,
+            "name": "{site_name}_ecommerce_client".format(
+                site_name=site_name,
+            ),
+            "url": url,
+            "client_type": CONFIDENTIAL,
+            "redirect_uri": "{url}complete/edx-oidc/".format(url=url),
+            "logout_uri": "{url}logout/".format(url=url)
+        }
```

Comment:
If you are passing all params as dict here, why not simply use keyword arguments?

Reply:
Basically the dict is converted to kwargs due to the `**` syntax, and I believe it looks prettier like this.

Reply:
Doesn't this look more readable/pretty?
```
Client.objects.get_or_create(
    user=service_user,
    name="{site_name}_ecommerce_client".format(
        site_name=site_name,
    ),
    url=url,
    client_type=CONFIDENTIAL,
    redirect_uri="{url}complete/edx-oidc/".format(url=url),
    logout_uri="{url}logout/".format(url=url)
)
```